vuln/npm/450.json

```diff
@@ -9,8 +9,8 @@
   "module_name": "eslint-scope",
   "cves": [],
   "vulnerable_versions": "3.7.2",
-  "patched_versions": "<=3.7.1 || >= 4.0.0",
-  "recommendation": "Use 3.7.1 for the latest 3.x major or upgrade to version 4.0.0 and above",
+  "patched_versions": "<=3.7.1 || >= 3.7.3",
+  "recommendation": "Use 3.7.1 for the latest 3.x major or upgrade to version 4.0.0 and above. It is advised that you revoke all of your existing npm tokens, recreate new tokens with 2FA enabled, and use lock files to pin down versions.",
   "references": [
```

Comment:
Perhaps also add `https://docs.npmjs.com/getting-started/using-two-factor-authentication` ref at the bottom.